ISISNeutronMuon/MDANSE
# MDANSE Trajectory and analysis extension update

**Description of work**
Updated MDANSE so the trajectory HDF files now have the extension .mdt and analysis hdf files now have the extension .mda. Updated the GUI so that the default filter is set to *.mdt. Added sandbox to gitignore.

**To test**
Run MDANSE unit test to ensure passing. Run conversion and analysis scripts to check the correct extensions are written. Test with the GUI.

![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/c8b7285f-52ac-4ba8-bf34-996f8326082b)

![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/1dff6c6b-0d79-4d5d-a8fb-ebc698215805)


closes #221
